common/presenters/filters-to-categories-list-component.js

```diff
@@ -1,5 +1,11 @@
+const queryString = require('qs')
+
 const i18n = require('../../config/i18n')
-const request = require('../lib/request')
+
+function getUrl(page, args) {
+  const stringified = queryString.stringify(args, { arrayFormat: 'comma' })
+  return `${page}${stringified ? `?${stringified}` : ''}`
+}
```

Comment:
I am slightly confused by this - I can't see this function being deleted from the `request` lib, but it looks like it's been moved to here? Was that because it wasn't used anywhere else?

Reply:
So, the method from the request lib is still being used elsewhere using the default settings so I didn't want to change that. However, in this file, switching to `qs` changed the default behaviour for handling values that are an array so needs to use `arrayFormat: 'comma'` to behave how it previously did.

I didn't want to get into re-writing this module too much with this change so I opted to having a private method to this module with that particular setting. 

Although whilst writing this, I wonder if the `getUrl` could support that option being passed in instead? What do you think?

Reply:
ah, that makes sense! I don't mind it being here and not using the request one, it's fine if it needs custom behaviour. And since this is the only place where the custom behaviour is used, I'd say there's limited value in refactoring the original method. Cheers! 👍